app/controllers/publishers_controller.rb

```diff
@@ -131,8 +131,9 @@ def verify
   # payment information.
   def verification_done
     @publisher = current_publisher
-    @publisher.uphold_state_token = SecureRandom.hex(64)
-    @publisher.save!
+    if @publisher.uphold_state_token.nil?
```

Comment:
@ayumi I'm not attached to this code, but we need some way to initialize the `uphold_state_token` for publishers who are already in the system, have already been verified, yet still need to connect with Uphold. This is related to the discussion we need to have about allowing publishers to switch Uphold accounts (and probably with newly generated state tokens).